Tests/Behat/FlowContextTrait.php

```diff
@@ -272,11 +254,6 @@ protected function resetFactories()
     protected function resetRolesAndPolicyService()
     {
         $this->objectManager->get(PolicyService::class)->reset();
-
-        if ($this->objectManager->isRegistered(RoleRepository::class)) {
```

Comment:
Why is this no longer needed?

Reply:
Because the RoleRepository doesn't exist since idk